packages/atomic/src/components/atomic-search-box/atomic-search-box.tsx

```diff
@@ -1,6 +1,15 @@
 import SearchIcon from 'coveo-styleguide/resources/icons/svg/search.svg';
```

Comment:
This whole file is much better ! 💯 

There is nothing specific for instant result, and everything is "generic", so it looks like it's going to be possible for our clients to be able to implement all kinds of advanced suggestion provider on their own.

Reply:
yeah, definitely!! I agree 🙂